cmd/call-api-endpoint/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Parameters []string `help:"Parameter for the endpoint in the format 'key:value', where the first colon separates the key and value (e.g., 'target-email:bob@test.edulinq.org'). Additional colons are allowed in the value." arg:"" optional:""`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("Execute an API request to the specified endpoint. For more information on available API endpoints, see the API resource file at: /autograder-server/resources/api.json"),
```

Comment:
The path is still off here.
Use an absolute path (and assuming the repo is at root) is no good.
Just use `resources/api.json` and assume you are at the repo's root (not filesystem root).